dev/NavigationView/NavigationView.xaml

```diff
@@ -284,7 +283,8 @@
                                                     ItemContainerStyle="{TemplateBinding MenuItemContainerStyle}"
                                                     ItemContainerStyleSelector="{TemplateBinding MenuItemContainerStyleSelector}"
                                                     SingleSelectionFollowsFocus="False"
-                                                    IsItemClickEnabled="True">
+                                                    IsItemClickEnabled="True"
+                                                    ElementSoundMode="{TemplateBinding ElementSoundMode}">
```

Comment:
I think this should just get picked up from the Flyout - were you seeing something different?

Reply:
Yeah, I was seeing the ancestor treewalk terminate at the ListView (just before the flyout). Seems to be because the walk only evaluates UIElements.